packages/parcel-transformer/src/index.ts

```diff
@@ -100,6 +100,8 @@ export default new Transformer<ParcelTransformerOpts>({
 
     const program = await parseAsync(code, {
       filename: asset.filePath,
+      babelrc: false,
```

Comment:
To be consistant, can you do the same for webpac-loader?

Reply:
sure